app/_layouts/docs-ee.html

```diff
@@ -0,0 +1,142 @@
+      <img class="page-header-bg-image-left" src="/assets/images/docs/bg-left.png" alt="left-bg-image">
+      <img class="page-header-bg-image-right" src="/assets/images/docs/bg-right.png" alt="right-bg-image">
+    </div>
+    {% endif %}
+    <div class="container">
+      <div class="edit-this-page enterprise">
+        <a target="_blank" href="https://github.com/Kong/docs.konghq.com/tree/master/app/{{page.path}}">
+          Edit this Page
+        </a>
+      </div>
+      <nav class="docs-navigation enterprise">
+        <a href="/">Kong</a>
+        <a href="/enterprise" class="active">Kong Enterprise</a>
+        <a href="/studio">Kong Studio</a>
+        <span class="sidebar-toggle">Open Sidebar</span>
+      </nav>
+
+      <div class="title-version-search-container">
+        <div class="page-header-title">
+          <h1>{{page.title}}</h1>
+        </div>
+        <div class="search-version-row">
+          <div class="versions-dropdown dropdown no-right-border">
+            <button class="page-header-btn" id="version-dropdown" type="button" data-toggle="dropdown" aria-haspopup="true"
+                    aria-expanded="false">
+              Version {{page.kong_version}}
+              {% if page.kong_version == "1.3-x" %}(2020){% endif %}
+              {% if page.kong_version == page.kong_latest.release %}<em>(latest)</em>{% endif %}
+              <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu dropdown-menu-right" role="menu" aria-labelledby="version-dropdown">
+              {% for ver in page.kong_versions reversed %}
+              <li {% if page.kong_version==ver.release %} class="active" {% endif %}>
+                <a href="/enterprise/{{ver.release}}" {% if ver.release==page.kong_version %} class="active" {% endif %}>
+                  {{ver.release}}
+                  {% if ver.release == "1.3-x" %}(2020){% endif %}
+                  {% if ver.release == page.kong_latest.release %}<em>(latest)</em>{% endif %}
+                </a>
+              </li>
+              {% endfor %}
+            </ul>
+          </div>
+          <div class="dropdown" id="searchBar">
+            <div id="getkong-algolia-search" class="search-input enterprise {% unless is_homepage %}small{% endunless %}">
+              <img class="icon" src="/assets/images/icons/icn-magnifying-glass.svg" alt="search">
+              <input type="text" placeholder="Type to search {% if is_homepage %}documentation...{% endif %}" id="getkong-algolia-search-input" />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <div class="container">
+    {% include docs-sidebar.html %}
+
+    {% if page.toc == true or page.toc != false and filename != "index.md" and is_homepage == false %}
+    <aside class="docs-toc">
+      <div class="docs-toc-title">On this page</div>
+      {% include toc.html html=content anchor_class="scroll-to" h_min=2 h_max=5 %}
```

Comment:
```suggestion
      {% include toc.html html=content anchor_class="scroll-to" h_min=2 h_max=3 %}
```
To clear the noise from lower headers